Leanplum-SDK/Classes/Features/Actions/LPActionContext.m

```diff
@@ -313,6 +310,17 @@ - (NSURL *)htmlWithTemplateNamed:(NSString *)templateName
     return nil;
 }
 
+-(NSString *)htmlStringContentsOfFile:(NSString *)file {
+    NSError *error;
+    NSString *htmlString = [NSString stringWithContentsOfFile:file
+                                                     encoding:NSUTF8StringEncoding
+                                                        error:&error];
+    if (error) {
```

Comment:
Maybe printing the error here or something that will show us why this method is returning nil